lib/generators/decidim/install_generator.rb

```diff
@@ -38,7 +38,14 @@ def copy_migrations
       end
 
       def add_seeds
-        append_file "db/seeds.rb", "Decidim::Core::Engine.load_seed"
+        append_file "db/seeds.rb" do
+          "\n# Loads seeds from all engines.
+Rails.application.railties.select do |railtie|
```

Comment:
I don't think it looks crappy. I'll accept also filtering by the engine name (and only using Decidim ones).